scc/git.py

```diff
@@ -1746,6 +1746,13 @@ def rmerge(self, filters, info=False, comment=False, commit_id="merge",
                set_commit_status=False, allow_empty=True, is_submodule=False):
         """Recursively merge PRs for each submodule."""
 
+        if self.repository_config is not None and \
+           "base-branch" in self.repository_config:
+            self.log.info("Overriding base-branch from %s to %s" %
```

Comment:
To reduce the logging verbosity, this could either be moved at the `debug` level and/or an extra check could be performed to skip the case where `filters["base"] == self.repository_config["base-branch"]`

Reply:
Updated to only warn when there's a change from the default.